configs/openmrs_config/ampathforms/health_center_NCD_Screening.json

```diff
@@ -328,6 +328,30 @@
                             },
                             "type": "obs",
                             "validators": []
+                        },
+                        {
+                            "id": "HBA1c",
+                            "questionOptions": {
+                                "rendering": "number",
+                                "concept": "a8af7520-1350-11df-a1f1-0026b9348838",
+                                "max": "20",
+                                "min": "0",
+                                "showDate": "false"
+                            },
+                            "type": "obs",
+                            "validators": []
+                        },
+                        {
+                            "id": "OGTT",
+                            "questionOptions": {
+                                "rendering": "number",
+                                "concept": "163594AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                                "max": "450",
+                                "min": "20",
```

Comment:
I may not really tell what should be the min and max value of the concept. Are these values of OGTT derived from computation of certain patient parameters(data)?

Reply:
If you search by the Uuid for that concept in question the values are given.